diff --git a/node_modules/@types/lowdb/index.d.ts b/node_modules/@types/lowdb/index.d.ts
index 40e6803..980ef36 100644
--- a/node_modules/@types/lowdb/index.d.ts
+++ b/node_modules/@types/lowdb/index.d.ts
@@ -256,7 +256,7 @@ declare namespace Lowdb {
   }

   interface lowdb {
-    <AdapterT extends AdapterAsync>(adapter: AdapterT): Promise<LowdbAsync<AdapterT[ReferenceProperty]>>;
+    <AdapterT extends AdapterAsync>(adapter: AdapterT):         LowdbAsync<AdapterT[ReferenceProperty]>;
     <AdapterT extends AdapterSync>(adapter: AdapterT): LowdbSync<AdapterT[ReferenceProperty]>;
   }
