diff --git a/node_modules/express-oas-generator/index.js b/node_modules/express-oas-generator/index.js
index 3a4a287..66e3201 100644
--- a/node_modules/express-oas-generator/index.js
+++ b/node_modules/express-oas-generator/index.js
@@ -170,8 +170,6 @@ module.exports.init = (aApp, aPredefinedSpec, aPath, aWriteInterval, aApiDocsPat
     next();
   });
 
-  // make sure we list routes after they are configured
-  setTimeout(() => {
     // middleware to handle requests
     app.use((req, res, next) => {
       try {
@@ -188,7 +186,6 @@ module.exports.init = (aApp, aPredefinedSpec, aPath, aWriteInterval, aApiDocsPat
       next();
     });
     init(aApiDocsPath);
-  }, 1000);
 };
 
 module.exports.getSpec = () => {
